bin/main.ml

```diff
@@ -426,6 +426,36 @@ let runtest =
           $ Arg.(value & pos_all string ["."] name_))
   , Term.info "runtest" ~doc ~man)
 
+let clean =
+  let doc = "Clean the project." in
+  let man =
+    [ `S "DESCRIPTION"
+    ; `P {|Removes files added by jbuilder including _build and .merlin|}
+    ; `Blocks help_secs
+    ]
+  in
+  let go common =
+    set_common common ~targets:[];
+    let log = Log.create () in
+    Future.Scheduler.go ~log
+      begin
+        (* Attach this first before we run anything else *)
+        at_exit (fun () ->
```

Comment:
Because there's another at_exit handler which dumps a a trace file called .db

Since the at_exit handlers are processed LIFO, I need this one to get in the stack first.